components/v1/HomePage/Navigation.tsx

```diff
@@ -27,18 +64,121 @@ const Navigation: FunctionComponent<NavigationProps> = ({}) => {
+                                                        <rect width="15" height="15" fill="white" />
+                                                    </clipPath>
+                                                </defs>
+                                            </svg>
+                                        </a>
+                                    </Popover.Panel>
+                                </Transition>
+
+                            </>
+                        )
+                    }}
+                </Popover>
+                
+                <Popover>
+                    {({open}) => {
+                        return(
+                            <>
+                                <Popover.Button ref={setReferenceElement2} className={`flex items-center text-sm ${open ? "text-gray-light-10 dark:text-gray-dark-10" : "text-gray-light-12 dark:text-gray-dark-12"} sm:self-center transition`}>
+                                    Resources 
+                                    <svg className={`${open ? "stroke-gray-light-10 dark:stroke-gray-dark-10" : "stroke-gray-light-12 dark:stroke-gray-dark-12"}`} xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24" fill="none" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round">
+                                        <path d="M6 9l6 6 6-6"/>
+                                    </svg>
+                                </Popover.Button>
+                                <Transition
+                                    enter="transition ease-out duration-133"
+                                    enterFrom="opacity-0"
+                                    enterTo="opacity-100"
+                                    leave="transition ease-in duration-100"
+                                    leaveFrom="opacity-100"
+                                    leaveTo="opacity-0"
+                                >
+                                    <Popover.Panel ref={setPopperElement2} style={popper2.styles.popper} {...popper2.attributes.popper} className="min-w-[161px] bg-gray-light-2 dark:bg-gray-dark-2 border border-gray-light-4 dark:border-gray-dark-4 flex flex-col p-[8px] rounded-[16px]">
+                                        <a href={RisedleLinks.github} className="flex items-center text-sm p-[8px] rounded-[8px] text-gray-light-12 dark:text-gray-dark-12 hover:bg-gray-light-3 hover:dark:bg-gray-dark-3 transition">
+                                            <span className="grow text-left">
+                                                Github <span className="text-gray-light-10 dark:text-gray-dark-10">&#8599;</span>
+                                            </span>
+                                            <svg width="15" height="16" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg" className="fill-gray-light-12 dark:fill-gray-dark-12 inline-block">
+                                                <path fill-rule="evenodd" 
```

Comment:
@math-marcellino u need to update `fill-rule` to `fillRule` & `clip-rule` to `clupRule`.

U can check if it valid by runing `npm run deploy` on your local machine

Reply:
Okay thank you for the review @pyk ! The fix has been committed and can be reviewed again.